### PR TITLE
Added content for about section and enhacement in design for same.

### PR DESCRIPTION
**Pull request by aashi2912**
## Related Issuse

About section is all empty. Only contains one line. Want to add more in about section related to website. What is represents and the features, use, developed by and other information.

Closes: #230 

#### Describe the changes you've made

Chane in 2 files. index.html and style.css
Add descriptive info about website in about section and little enhancement in design/style for same.
We can do a lot better and more in design still but considering the current scenario and the overall color and UI theme of whole website, This is all I came up with right now. Whole color, theme and UI needs to be changed in future for further improvement.

## Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.

## Screenshots

aboutus makesmatheasy

Review this and tell me if anything more you would like to do or any other change or modification. @shrey27tri01

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2272,12 +2272,29 @@ <h2 style="color: var(--appimage);">Simple and Compund Interest</h2>
                 <table class="table">
                     <tr>
                         <td>
-                            <h5>App is Developed By <span class="badge badge-light" style="cursor:pointer;"
-                                    onclick="window.open('https://sairish2001.github.io')">Sairish</span>
+                            <h5>MakesMathEasy Solves various Math Problems along with Steps. We are providing Complex
+                                Calculations within a few milliseconds along with speed. We work together to
+                                design, create and produce work that we are proud of. We believe that <span
```
Comment (shrey27tri01) [inline index.html:2277]: This about info is great! Just a few changes I'd like to suggest:

> MakesMathEasy solves various math problems along with steps. We provide speedy calculations, within a few. We work together, design, create and produce work that we are proud of. We believe that

The "MATH IS FUN!" part is great, no need to change that :)

**Review by shrey27tri01 — CHANGES_REQUESTED** [review]
Awesome work! I just suggested some minor changes, please have a look at the inline review, and then we can merge this!

**Comment by aashi2912** [comment]
Sorry. I didn't understand how this works as I am new to this. You want this style.css files underline and shadow thing back??

I removed it because it looks like link white underline and shadow thing too.

And also how should I resolve this and procedure further . Please guide me :)

**Comment by aashi2912** [comment]
or you just said to change this paragraph like this as u mentioned right?

MakesMathEasy solves various math problems along with steps. We provide speedy calculations, within a few. We work together, design, create and produce work that we are proud of. We believe that

**Comment by shrey27tri01** [comment]
> or you just said to change this paragraph like this as u mentioned right?

@aashi2912 Exactly! I wanted you to change that paragraph. 
That is what we call as an "inline review", where I requested you to make some changes in the pull request you created as a part of my review.

You can read more about these things [here](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-request-reviews) and [here](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request) :smile:

**Comment by aashi2912** [comment]
Okk.. will do it. Thank you 😄!

**Comment by aashi2912** [comment]
Updated the file and commit the change.😅

File: index.html (hunk 1 above)
Comment (shrey27tri01) [reply index.html]: Great job!

**Review by shrey27tri01 — APPROVED** [review]
Awesome work on this!

> Thank you

No worries!

File: index.html (hunk 1 above)
Comment (aashi2912) [reply index.html]: TYSM😄

**Pull request merged**